strawberryfields/backends/bosonicbackend/bosoniccircuit.py

```diff
@@ -116,7 +116,7 @@ def del_mode(self, modes):
             self.loss(0.0, mode)
             self.active[mode] = None
 
-    def reset(self, num_subsystems=None, num_weights=None):
+    def reset(self, num_subsystems=None, num_weights=1):
```

Comment:
In the docstrong in line 124 you might want to write `None` instead of None.

Reply:
Will do!

Reply:
Done